extensions/pkg/controller/worker/templates/machineclasses.tpl.yaml

```
@@ -14,6 +14,8 @@ spec:
     listKind: MachineClassList
     plural: machineclasses
     singular: machineclass
+    shortNames:
+    - mcc
```

Cosmetic suggestion to reduce no-op diff compared to the original CRDs in machine-controller-manager:
```suggestion
    shortNames:
    - mcc
    singular: machineclass
```